extensions/aws/processors/S3Processor.h

```diff
@@ -83,23 +90,67 @@ struct CommonProperties {
+      .withDescription("The S3 bucket")
+      .isRequired(true)
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto AccessKey = core::PropertyDefinitionBuilder<>::createProperty("Access Key")
+      .withDescription("AWS account access key")
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto SecretKey = core::PropertyDefinitionBuilder<>::createProperty("Secret Key")
+      .withDescription("AWS account secret key")
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto CredentialsFile = core::PropertyDefinitionBuilder<>::createProperty("Credentials File")
+      .withDescription("Path to a file containing AWS access key and secret key in properties file format. Properties used: accessKey and secretKey")
+      .build();
+  EXTENSIONAPI static constexpr auto AWSCredentialsProviderService = core::PropertyDefinitionBuilder<>::createProperty("AWS Credentials Provider service")
+      .withDescription("The name of the AWS Credentials Provider controller service that is used to obtain AWS credentials.")
+      .build();
+  EXTENSIONAPI static constexpr auto Region = core::PropertyDefinitionBuilder<region::REGIONS.size()>::createProperty("Region")
+      .isRequired(true)
+      .withDefaultValue(region::US_WEST_2)
+      .withAllowedValues(region::REGIONS)
+      .withDescription("AWS Region")
+      .build();
+  EXTENSIONAPI static constexpr auto CommunicationsTimeout = core::PropertyDefinitionBuilder<>::createProperty("Communications Timeout")
+      .isRequired(true)
+      .withPropertyType(core::StandardPropertyTypes::TIME_PERIOD_TYPE)
+      .withDefaultValue("30 sec")
+      .withDescription("Sets the timeout of the communication between the AWS server and the client")
+      .build();
+  EXTENSIONAPI static constexpr auto EndpointOverrideURL = core::PropertyDefinitionBuilder<>::createProperty("Endpoint Override URL")
+      .withDescription("Endpoint URL to use instead of the AWS default including scheme, host, "
+          "port, and path. The AWS libraries select an endpoint URL based on the AWS "
+          "region, but this property overrides the selected endpoint URL, allowing use "
+          "with other S3-compatible endpoints.")
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto ProxyHost = core::PropertyDefinitionBuilder<>::createProperty("Proxy Host")
+      .withDescription("Proxy host name or IP")
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto ProxyPort = core::PropertyDefinitionBuilder<>::createProperty("Proxy Port")
+      .withDescription("The port number of the proxy host")
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto ProxyUsername =   core::PropertyDefinitionBuilder<>::createProperty("Proxy Username")
```

Comment:
```suggestion
  EXTENSIONAPI static constexpr auto ProxyUsername = core::PropertyDefinitionBuilder<>::createProperty("Proxy Username")
```

Reply:
fixed in bde74b220d3a6755b91e5104648b0a2b0f57fafc (also the other `=  ` issues below)